src/core/SaleorSDK.ts

```diff
@@ -1,21 +1,23 @@
+import { ApolloClient, NormalizedCacheObject } from "@apollo/client";
 import { auth } from "./auth";
-import { createSaleorClient } from "../apollo";
-import { Core, SaleorSDKConfig } from "./types";
+import { Core } from "./types";
 
-export const SaleorSDK = ({ apiUrl, channel }: SaleorSDKConfig): Core => {
+export const SaleorSDK = (
+  client: ApolloClient<NormalizedCacheObject>,
+  channel: string
+): Core => {
   let _channel = channel;
+
   const setChannel = (channel: string): string => {
     _channel = channel;
     return _channel;
   };
 
-  const apolloClient = createSaleorClient(apiUrl);
-  const coreInternals = { apolloClient, channel: _channel };
-  const authSDK = auth(coreInternals);
+  const authSDK = auth(client, channel);
```

Comment:
```
const apolloClient = createSaleorClient(apiUrl);
const coreInternals = { apolloClient, channel: _channel };
const authSDK = auth(coreInternals);
```
The old logic was fine here but `createSaleorClient` should be called `createApolloClient` because that's exactly what it does.